apps/settings/locales/settings.en-US.properties

```diff
@@ -74,8 +76,9 @@ wifi-bad-credentials-confirm=Unable to connect to {{ssid}}, authentication faile
 # in headers, you can use a shorter string here.
 manageNetworks-header={{manageNetworks}}
 manageCertificates=Manage certificates
-manageCert=Manage Cert.
+manageCert=Manage Certificates
```

Comment:
Here's another case were you can't change a string like this.  I'm going to stop commenting on these, but it is a general rule that you can't do this.  

In this particular case, it seems clear to me that our UI needed a very short string for something to fit. So by making the string longer, you probably broke something.